docs/config/settings.md

```diff
@@ -268,3 +268,37 @@ See the NGINX documentation in addition.
 If you use OpenStreetMap as OAuth 2 provider, you can use those settings.
 
 Otherwise, use any valid [python-social-auth configuration](https://python-social-auth.readthedocs.io/en/latest/configuration/django.html).
+
+#### WEBSOCKET_ENABLED
+
+A websocket server is packaged with uMap, and can be turned-on to activate "real-time collaboration".
+In practice, you would need to run the websocket server and specify a set of settings.
+
+Turning this setting to `True` **will make a switch available** on each map, to enable "real-time collaboration".
+
+You can run the websocket server with this command:
+
+```bash
+python -m umap.ws
+```
+
+Configuration example:
+
+```python
+WEBSOCKET_ENABLED = True
+WEBSOCKET_HOST = "localhost"
+WEBSOCKET_PORT = 8002
+WEBSOCKET_URI = "ws://localhost:8002"
+```
+
+These settings can also be set with the (same names) environment variables.
+
+#### WEBSOCKET_HOST
+#### WEBSOCKET_PORT
+
+The host and port for the websocket server.
+
+#### WEBSOCKET_URI
+
+The connection string that will be used by the client to connect to the websocket server.
+Use `wss://host:port` if the server is behind TLS, and `ws://host:port` otherwise.
```

Comment:
I see why we would like this, as it would avoid duplicating the information in the settings.

We need to take in consideration that the "front" WebSocket address (that clients will connect to) might be different than the "back" ip and port which are bound in the host.

This happens for instance for reverse proxies, or when running inside a container. In this case, magically guessing the "front" address based on `WEBSOCKET_HOST`, `WEBSOCKET_PORT` and `WEBSOCKET_TLS` might not be enough, and we would need to introduce a `WEBSOCKET_URI` setting like the one that's already present in this PR.

I've changed the settings to reflect this `BACK`/`FRONT` difference, and updated the documentation to make it clearer, see commit [`0757d3f` (#1754)](https://github.com/umap-project/umap/pull/1754/commits/0757d3f0c7c97a7c13e9016ccb8f625d7ca46141)